client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -28,6 +29,18 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = ( {
 		case Steps.IntentGathering:
 			return (
 				<div className="onboarding-block__acquire-intent">
+					<SwitchTransition mode="in-out">
```

Comment:
The `in-out` mode is adding the following child to the DOM while still keeping the previous one for the duration of the timeout so we can transition the opacity for both to achieve the cross-fade effect